packages/ERTP/test/unitTests/test-payment-as-extent-attack.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * This file shows that using a payment as the extent of another payment is a bad idea
+ * 
+ * For this purpose, we have Alice who initially owns 1000 Money (which is a fungible asset)
+ * She wants to buy a Vault from Joker for 100 Money. This Vault contains 1 Apple payment
+ * 
+ * The Vault is a non-fungible asset. Apples are fungible assets
+ *  
+ * A Vault is a payment. The extent of the Vault is an object with a single `open` method
+ * This open method returns whatever is in the Vault
+ * 
+ * Joker will create the Vault
+ * 
+ */
+
+import { assert } from "@agoric/assert";
+
+// simple substitute for Zoe + SimpleExchange contract
+function makeFakeSimpleExchange() {
```

Comment:
Do you think we could make this an ERTP example only and remove the fake Zoe contract? I think that would be a lot simpler to understand, but let me know if you think it would removing an important part of the example.